modules/acm_sku/src/ProductManager.php

```diff
@@ -422,19 +422,6 @@ public function synchronizeProducts(array $products = [], $storeId = '') {
           continue;
         }
 
-        $query = $this->entityManager->getStorage('acm_sku')->getQuery()
-          ->condition('sku', $product['sku']);
-        $sku_ids = $query->execute();
-
-        if (count($sku_ids) > 1) {
-          $this->failedSkus[] = $product['sku'] . '(Duplicate product SKU found.)';
```

Comment:
not sure why would we fail product update, removed this code to ensure we can update products